modules/cnf-topology-aware-lifecycle-operator-installation-cli.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-talo-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="installing-topology-aware-lifecycle-operator-using-cli_{context}"]
+= Installing the Topology Aware Lifecycle Operator by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the Topology Aware Lifecycle Operator.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of {rh-rhacm} Operator.
+* Set up a Hub cluster with disconnected registry.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `Namespace` custom resource (CR) for the Topology Aware Lifecycle Operator:
+.. Define the `Namespace` CR and save the YAML file, for example, `talo-namespace.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-cluster-group-upgrades
+----
+.. Create the `Namespace` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f talo-namespace.yaml
+----
+
+. Create an `OperatorGroup` CR:
+.. Define the `OperatorGroup` CR and save the YAML file, for example, `talo-group.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: talo
+  namespace: openshift-operators
+spec:
+  targetNamespaces:
+  - openshift-operators
+----
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f talo-group.yaml
+----
+
+. Create a `Subscription` CR:
+.. Define the `Subscription` CR and save the YAML file, for example, `talo-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: talo
+    namespace: openshift-operators
+spec:
+    channel: alpha
+    name: talo
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+    package: talo
+----
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f talo-subscription.yaml
+----
+
+.Verification
+
+. Verify that the installation succeeded by inspecting the CSV resource:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-operators
+----
++
+.Example output
+[source,terminal]
+----
+NEEDED
+----
+
+. Verify that the Topology Aware Lifecycle Operator is up and running:
++
+[source,terminal]
+----
+$ oc get deploy -n openshift-operators
+----
++
+.Example output
+[source,terminal]
+----
+NEEDED
```

Comment:
NAMESPACE                                          NAME                                             READY   UP-TO-DATE   AVAILABLE   AGE
openshift-operators                                cluster-group-upgrades-controller-manager        1/1     1            1           14s